vignettes/orderly.Rmd

```diff
@@ -571,6 +571,38 @@ would look up the field `password` at the path
 `orderly_config.yml`, in the contents of an environment variable or
 in `orderly_envir.yml` (currently this only uses the vault [version 1 key-value storage](https://www.vaultproject.io/docs/secrets/kv/kv-v1.html))
 
+The above approach can be used to switch databases by using different environmental variables, but that can become tiresome.  If you have multiple database "instances" corresponding to different realisations of the same logical database (e.g., production and staging), then you can configure and switch between these by using the `instances` field.  List commmon arguments within the `args` block as before, then add named instances:
```

Comment:
please check to make sure the approach is explained clearly enough

Reply:
Looks fine to me,.